/pointcloud_to_laserscan:
  ros__parameters:
    angle_increment: 0.0034828970674425364
    angle_min: -3.1241390705108643
    angle_max: 3.1415927410125732
    inf_epsilon: 1.0
    max_height: 1.0
    min_height: 0.0
    qos_overrides:
      /parameter_events:
        publisher:
          depth: 1000
          # durability: volatile
          # history: keep_last
          reliability: best_effort
    queue_size: 16
    range_min: 0.15000000596046448
    range_max: 40.0
    scan_time: 0.049273036420345306
    target_frame: laser
    transform_tolerance: 0.01
    use_inf: true
    use_sim_time: false
    time_increment: 2.7389125534682535e-05



